modules/kubernetes/datadog/templates/values.yaml.tpl

```diff
@@ -0,0 +1,34 @@
+datadog:
+  apiKey: ${api_key}
+  clusterName: ${location}-${environment}
+  site: datadoghq.eu
```

Comment:
Make site a variable that defaults to datadoghq.eu

Reply:
sounds good